src/ring_theory/jacobson_ideal.lean

```diff
@@ -0,0 +1,121 @@
+/-
+Copyright (c) 2020 Devon Tuma. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Kenny Lau, Devon Tuma
+-/
+import ring_theory.ideal_operations
+
+universes u v
+
+namespace ideal
+variables {R : Type u} [comm_ring R]
+
+section jacobson
+
+/-- The Jacobson radical of `I` is the infimum of all maximal ideals containing `I`. -/
+def jacobson (I : ideal R) : ideal R :=
+Inf {J : ideal R | I ≤ J ∧ is_maximal J}
+
+lemma le_jacobson {I : ideal R} : I ≤ jacobson I :=
+λ x hx, mem_Inf.mpr (λ J hJ, hJ.left hx)
+
+theorem jacobson_eq_top_iff {I : ideal R} : jacobson I = ⊤ ↔ I = ⊤ :=
```

Comment:
This should be a `simp`. lemma, and you should prove `jacobson ⊤ = ⊤` and make it a `simp` lemma as well.